portfolio/src/main/webapp/script.js

```diff
@@ -73,7 +73,15 @@ function getComments() {
   fetch(url) 
   .then(response => response.json())
   .then((messages) => {
-    document.getElementById('message-container').innerText = messages.join('\n');
+    document.getElementById('message-container').innerHTML = '';
+
+    for (message of messages) {
+      let commentContainer = document.createElement('div');
+      commentContainer.id = 'comment-container';
```

Comment:
This makes it such that multiple comment elements have the same id. IDs should be unique so they could be queried easily. Instead of id, you should use class.